### PR TITLE
Feat: LCFS - IDIR users can filter transactions by organization#192

### PR DESCRIPTION
**Pull request by prv-proton**
Depends on #190

### CODE REVIEW

**Comment by prv-proton** [comment]
Depends on #190

File: backend/lcfs/web/api/organization/views.py [hunk 1]
```diff
@@ -333,3 +363,70 @@ async def get_users_for_organization(
...
+    organization_id: int,
+    db: AsyncSession = Depends(get_async_db),
+    pagination: PaginationRequestSchema = Body(..., embed=False),
+    response: Response = None,
+):
+    try:
+        offset = 0 if (pagination.page < 1) else (pagination.page - 1) * pagination.size
+        limit = pagination.size
+
+        query = (
```
Comment (hamed-valiollahi) [inline backend/lcfs/web/api/organization/views.py:379]: Should we move the queries to session.py?

File: backend/lcfs/web/api/transaction/session.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+            return [
+                Transaction.model_validate(transaction) for transaction in results
+            ], total_count
+
+        except Exception as e:
+            logger.error(f"Error occurred while fetching transactions: {e}")
+            raise Exception(f"Error occurred while fetching transactions")
+
+    async def export_transactions(
```
Comment (hamed-valiollahi) [inline backend/lcfs/web/api/transaction/session.py:101]: Should this part be moved to views.py?

**Review by hamed-valiollahi — COMMENTED** [review]
Great job! I just noticed some issues with the table columns when viewed on different screen sizes.

**Review by hamed-valiollahi — APPROVED** [review]

File: backend/lcfs/web/api/organization/views.py (hunk 1 above)
Comment (prv-proton) [reply backend/lcfs/web/api/organization/views.py:379]: Thanks for the comment. I think these changes are from Kevin's branch. I'll try to rebase mine with his after his branch is merged.

File: backend/lcfs/web/api/transaction/session.py (hunk 2 above)
Comment (prv-proton) [reply backend/lcfs/web/api/transaction/session.py:101]: Thanks for the comment. I think these changes are from Kevin's branch. I'll try to rebase mine with his after his branch is merged.

File: backend/lcfs/web/api/organization/views.py (hunk 1 above)
Comment (hamed-valiollahi) [reply backend/lcfs/web/api/organization/views.py:379]: No problem! Thanks for letting me know, Prashanth!

**Pull request closed without merging**